src/app/app.component.ts

```diff
@@ -1,14 +1,34 @@
-import { Component } from '@angular/core';
+import { Component, ElementRef, OnInit, ViewChild } from '@angular/core';
+import Map from 'arcgis-js-api/Map';
+import MapView from 'arcgis-js-api/views/MapView';
 import { Observable } from 'rxjs';
 
 @Component({
   selector: 'agrc-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.styl']
 })
-export class AppComponent {
+export class AppComponent implements OnInit {
   findResult: string;
   apiKey = 'AGRC-16E793E2220676';
+  quadWord = '??';
```

Comment:
we should use file replacements like dts does in the business portal.

https://angular.io/guide/workspace-config#additional-build-and-test-options